src/components/navbar/navLinkSection/NavLinkSection.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { Fragment } from 'react';
+import { Link } from 'react-router-dom';
+import classes from './NavLinkSection.module.scss';
+import PropTypes from 'prop-types';
+
+const NavLinkSection = ({ navLink, closeDropdownNav, toggleNewsModal }) => {
+    return (
+        <Fragment>
+            {navLink.map(({ display, path, external, modal }) => {
+                return path ? (
+                    <Link
+                        key={display}
+                        className={classes.link}
+                        to={path}
+                        onMouseDown={closeDropdownNav}
+                    >
+                        {display}
+                    </Link>
+                ) : external ? (
+                    <a
+                        key={display}
+                        className={classes.link}
+                        href={external}
+                        target='_blank'
+                        rel='noopener noreferrer'
+                        onMouseDown={closeDropdownNav}
+                    >
+                        {display}
+                    </a>
+                ) : modal ? (
+                    <a
+                        key={display}
+                        className={classes.link}
+                        rel='noopener noreferrer'
+                        onMouseDown={toggleNewsModal}
```

Comment:
Is there a positive to using `onMouseDown` everywhere over the standard `onClick`?

Reply:
there was an issue of mouse down changing state, holding it kept the state, then mouse up changed the state again. This problem became pretty consistent with the dropdown burger icon.